package.json

```diff
@@ -11,7 +11,8 @@
     "build": "next build",
     "start": "next start",
     "generate-css-types": "tailwindcss-classnames -x ./tailwindcss-custom-types.ts -i ./tailwind.config.js",
-    "pre-commit": "lint-staged"
+    "pre-commit": "lint-staged",
+    "pre-push": "tsc --project tsconfig.json"
```

Comment:
We should probably use `npx` again incase someone doesn't have TS globally installed.

`npx tsc --project tsconfig.json`

Reply:
@bhgeorge  reminded me that since we're running these commands from the root where the modules are also installed, we shouldn't need `npx` since it's a package dependency.